### PR TITLE
VAEGAN_Tf

### PR DESCRIPTION
**Pull request by Umang-pandey**

### CODE REVIEW

File: generative_models/VAEGAN_Tensorflow/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+![](https://github.com/Umang-pandey/vae-gan-tf/blob/master/images/algorithm.png)
+
+# Architecture
+ 
+ ## Encoder
+  ![](https://github.com/Umang-pandey/vae-gan-tf/blob/master/images/generator.png)
+ ## Discriminator
+  ![](https://github.com/Umang-pandey/vae-gan-tf/blob/master/images/dicriminator.png)
+ ## Decoder
+  ![](https://github.com/Umang-pandey/vae-gan-tf/blob/master/images/decoder.png)
```
Comment (nirmal-suthar) [inline generative_models/VAEGAN_Tensorflow/README.md]: use local path ( ie. ./images/*)

File: generative_models/VAEGAN_Tensorflow/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Fashion MNIST 
+### After 500 Epochs
+
+![](https://github.com/Umang-pandey/vae-gan-tf/blob/master/images/fashion_mgif.gif)
+
+
+## MNIST
+### After 500 Epochs
+![](https://github.com/Umang-pandey/vae-gan-tf/blob/master/images/mnist_gif.gif)
+
```
Comment (nirmal-suthar) [inline generative_models/VAEGAN_Tensorflow/README.md]: same here

**Comment by nirmal-suthar** [comment]
- In README, use local path
- Rename images to assets, and do according changes to READE
- maybe split the main.py into separate files for better visibility like model.py, data.py, utils.py and main.py

**Review by dev-chauhan — APPROVED** [review]

**Comment by Umang-pandey** [comment]
I hope all the prerequisites of the pr completed. Is there anything else left?

**Pull request merged**